Microsoft.Azure.Cosmos/src/ChangeFeedHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
+namespace Microsoft.Azure.Cosmos
+{
+    using Microsoft.Azure.Documents;
+
+    internal static class ChangeFeedHelper
+    {
+        internal static bool isChangeFeedWithQueryRequest(
+           OperationType operationType,
+           bool hasStreamPayload = false)
+        {
+            if (operationType == OperationType.ReadFeed && hasStreamPayload)
```

Comment:
This could be simplified by just:

```suggestion
            return operationType == OperationType.ReadFeed && hasStreamPayload;
```